pnr/symbiflow-vtr-optimized/build.sh

```diff
@@ -0,0 +1,97 @@
+#!/bin/bash
+
+set -e
+set -x
+
+if [ x"$TRAVIS" = xtrue ]; then
+	CPU_COUNT=2
+fi
+
+echo "============================================================"
+echo "CFLAGS='$CFLAGS'"
+echo "CXXFLAGS='$CXXFLAGS'"
+echo "CPPFLAGS='$CPPFLAGS'"
+echo "DEBUG_CXXFLAGS='$DEBUG_CXXFLAGS'"
+echo "DEBUG_CPPFLAGS='$DEBUG_CPPFLAGS'"
+echo "LDFLAGS='$LDFLAGS'"
+echo "------------------------------------------------------------"
+# -Wundef causes warnings on undefined preprocessor defines (e.g. o
+# tbb_config.h:56:7: warning: "__GLIBCPP__" is not defined, evaluates to 0 [-Wundef]
+# elif __GLIBCPP__ || __GLIBCXX__
+export CFLAGS="$(echo $CFLAGS | sed -e's/-Wundef //') -w"
+export CXXFLAGS="$(echo $CXXFLAGS | sed -e's/-Wundef //') -w"
+export CPPFLAGS="$(echo $CPPFLAGS | sed -e's/-Wundef //')"
+export DEBUG_CXXFLAGS="$(echo $DEBUG_CXXFLAGS | sed -e's/-Wundef //') -w"
+export DEBUG_CPPFLAGS="$(echo $DEBUG_CPPFLAGS | sed -e's/-Wundef //')"
+echo "CFLAGS='$CFLAGS'"
+echo "CXXFLAGS='$CXXFLAGS'"
+echo "CPPFLAGS='$CPPFLAGS'"
+echo "DEBUG_CXXFLAGS='$DEBUG_CXXFLAGS'"
+echo "DEBUG_CPPFLAGS='$DEBUG_CPPFLAGS'"
+echo "LDFLAGS='$LDFLAGS'"
+env
+
+# Where to get data needed to collect profile data in run-sf.sh
+URL_BASE=https://storage.googleapis.com/symbiflow-arch-defs-gha/
+declare -A PACKAGE
+PACKAGE[benckmarks]=symbiflow-benchmarks-latest
+PACKAGE[toolchain]=symbiflow-toolchain-latest
+PACKAGE[architecture]=symbiflow-xc7a50t_test-latest
+
+BUILD_ROOT=$PWD
+
+# Executables needed for PGO
+PGO_TARGETS="vpr genfasm"
+
+# This will take a while to download, so fork it and rejoin later.
+mkdir symbiflow
+(
+    for data in "${!PACKAGE[@]}"; do
+        # Given URL points to another URL containing a .tar.xz
+        curl ${URL_BASE}${PACKAGE[$data]} | xargs curl | tar xJ -C symbiflow
+    done
+) &
+FETCH_SYMBIFLOW_PID=$!
+
+export M4=${PREFIX}/bin/m4
```

Comment:
What needs m4?

Reply:
This was copied from `symbiflow-vtr/build.sh`. I'll check if it is still necessary.

Reply:
I've removed this.